api/src/main/java/io/strimzi/api/kafka/model/listener/KafkaListenerExternalIngress.java

```diff
@@ -80,8 +80,7 @@ public void setConfiguration(IngressListenerConfiguration configuration) {
         this.configuration = configuration;
     }
 
-    @Description("Configures the `Ingress` class that defines which `Ingress` controller will be used. " +
-            "If not set, the `Ingress` class is set to `nginx`.")
+    @Description("Configures the `Ingress` class that defines which `Ingress` controller will be used.")
```

Comment:
does it mean that it's mandatory to be set now? what happens if not set and not default to nginx anymore?

Reply:
No, it is not mandatory now. But the default value in the past was `nginx`. Now it will have no default value and when not configured it will not be set at all (which in my case for example doesn't matter).

Reply:
I improved the description to make it more clear.